### DISCUSSION TITLE
How to Export/Import bookmarks?

### DISCUSSION THREAD

**Discussion by Uzver123** [WerWolv/ImHex#1589] [Feedback]
I parsed 10 similar binary files file with Python and generate bookmarks automatically now i want to import them into Hex editor for ease of working.

By design imHex limiting users to slowly making bookmarks inside the editor one-by-one and saving them in some encrypted `.hexproj` format that can't be edited easily with Notepad++.

Tried to change single letter in bookmark name and it will not load anymore after saving.

![image](https://github.com/WerWolv/ImHex/assets/143356096/d82c471a-27b0-4535-b860-687165467c82)

Is there way to quickly edit `.hexproj` i don't want to use Hex editor in order to modify bookmarks file...

**Comment by WerWolv** [comment]
What are you talking about? 
Bookmarks can be exported through `File -> Export -> Bookmark`. Also project files are quite literally just .tar files. You can open them with any decompression tool that supports tar files.

**Reply by Uzver123** [reply]
Fantastic info thanks, you made my day.
